c2rust-ast-exporter/src/AstExporter.cpp

```diff
@@ -2463,7 +2463,7 @@ class TranslateConsumer : public clang::ASTConsumer {
             }
 #else  // CLANG_VERSION_MAJOR >= 10
             const FileID file = sourceMgr.getMainFileID();
-            auto comments = Context.getRawCommentList().getCommentsInFile(file);
+            auto comments = Context.Comments.getCommentsInFile(file);
```

Comment:
If this is new to clang 11, it should be wrapped in a version check (see `CLANG_VERSION_MAJOR` above).

Reply:
New in clang 11 is the removal of the getter. The field is public since at least clang 6. Does that warrant an `#ifdef`?

Reply:
> Does that warrant an #ifdef?

No, that's fine, as long as the new code works on older versions.